README.md

```diff
@@ -167,6 +167,7 @@ an [issue](https://github.com/spacetelescope/romancal/issues).
 | 0.14.0    | 24Q2_B13  | 056          | Feb 2024  | Release for Build 24Q2_B13 (Build 13) |
 | 0.15.0    | 24Q3_B14  | 058          | May 2024  | Release for Build 24Q3_B14 (Build 14) |
 | 0.15.1    | 24Q3_B14  | 058          | May 2024  | Release for Build 24Q3_B14 (Build 14) |
+| 0.16.0    | 24Q4_B15  | 061          | Sep 2024  | Release for Build 24Q3_B15 (Build 15) |
```

Comment:
should the Date column be the release date (could be August?) or build date (September)?

Reply:
I think our release 0.16.0 should be when we put the release together, so Aug 2024
When our release is bundled with the SDF code they can use the Sept 2024 date. 
Also can you update the context from 061 --> 063